src/commands/commerce/store/quickstart/setup.ts

```diff
@@ -1014,4 +997,21 @@ export class StoreQuickstartSetup extends SfdxCommand {
         );
         await this.statusFileManager.setValue('updatedFlowAssociatedToCheckout', true);
     }
+
+    private async getBuyerProfileUsingTemplate(): Promise<String> {
+        let template;
+        if (!(template = await this.statusFileManager.getValue('template'))) return Promise.resolve(defaultProfile);
+        else {
+            this.ux.log(`Template is ${template}`);
```

Comment:
Do we want to pull this string out into the `messages` folder?

Reply:
These are for debugging